tests/integration/devfile/cmd_devfile_push_test.go

```diff
@@ -925,6 +925,47 @@ var _ = Describe("odo devfile push command tests", func() {
 		Context("odo push (without -f) is executed", func() {
 			ensureFilesSyncedTest(commonVar.Project, false)
 		})
+
+		When("node-js application is created and pushed with devfile schema 2.2.0", func() {
```

Comment:
```suggestion
		When("node-js application is created with devfile schema 2.2.0", func() {
```

Reply:
I think we should change this as this are the steps that we are doing in the before each.
From my perspective, `When` should inform the steps taken in the `beforeEach`.

Reply:
My bad, I didn't read the code correctly.